.github/workflows/build.yaml

```diff
@@ -11,14 +11,19 @@
 #  SPDX-License-Identifier: Apache-2.0
 # *******************************************************************************/
 
-name: Build checks and tests
+name: Lint and Test
+
+env:
+  VSOMEIP_INSTALL_PATH: vsomeip-install
+  GENERIC_CPP_STDLIB_PATH: /usr/include/c++/11
+  ARCH_SPECIFIC_CPP_STDLIB_PATH: /usr/include/x86_64-linux-gnu/c++/11
```

Comment:
int the future it might be beneficial to create symlinks so that the runner applies the right path rather than hardcoded version dependent ones.

Reply:
Opened an issue for this, marking this as resolved for now